homeassistant/data_entry_flow.py

```diff
@@ -694,7 +695,7 @@ def add_suggested_values_to_schema(
             ):
                 # Copy the marker to not modify the flow schema
                 new_key = copy.copy(key)
-                new_key.description = {"suggested_value": suggested_values[key]}
+                new_key.description = {"suggested_value": suggested_values[key.schema]}
```

Comment:
Was this a bug?

Reply:
No. The `hash` from `key` is delegated to `key.schema` for `vol.Marker` types. Mypy doesn't know that though.